EXILED.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.30225.117
+# Visual Studio 15
+VisualStudioVersion = 15.0.28307.1300
```

Comment:
This is not needed.

Reply:
Sorry, this was done automatically by my VisuAL sTUDIO. i GOTTA UPDATE TO 2019 LOL

Reply:
No problem, but you still have to revert it and the things I pointed out above.

Reply:
Yeah sorry I'll do that right now